src/main/java/frc/robot/platforms/DarrylMini.java

```diff
@@ -0,0 +1,43 @@
+package frc.robot.platforms;
+
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+import frc.robot.subsystems.DrivetrainSRX;
+import frc.robot.subsystems.MotorSRX;
+
+public class DarrylMini implements RobotRunnable {
+    
+    MotorSRX dmotor;
+    final XboxController driveHID;
+    final CommandXboxController driveController;
+    public DarrylMini() {
```

Comment:
Put member variables right after the class name. Leave a space before the constructor.